runtime/src/bank.rs

```diff
@@ -4262,12 +4276,14 @@ impl Bank {
                 .feature_builtins
                 .extend_from_slice(&additional_builtins.feature_builtins);
         }
-        for builtin in builtins.genesis_builtins {
-            self.add_builtin(
-                &builtin.name,
-                builtin.id,
-                builtin.process_instruction_with_context,
-            );
+        if !debug_do_not_add_builtins {
```

Comment:
this is what we need to lobotomize so we don't try to add builtin accounts that normally would already exist in a full account set load.